src/apiNetworkProvider.ts

```diff
@@ -36,7 +36,7 @@ export class ApiNetworkProvider implements INetworkProvider {
     }
 
     private getProxyConfig(config: NetworkProviderConfig | undefined) {
-        let proxyConfig = JSON.parse(JSON.stringify(config));
+        let proxyConfig = JSON.parse(JSON.stringify(config || {}));
```

Comment:
maybe `??` instead of `||`

> the `||` operator is used to provide a default value when a variable is falsy, whereas the `??` operator is used to provide a default value when a variable is null or undefined